enterprise/security/src/main/java/org/neo4j/server/security/enterprise/auth/AuthProcedures.java

```diff
@@ -405,6 +405,13 @@ public void deleteRole( @Name( "roleName" ) String roleName )
         }
     }
 
+    @Description( "Clears authentication cache." )
+    @Procedure( name = "dbms.security.clearAuthenticationCache", mode = DBMS )
+    public void clearAuthenticationCache()
```

Comment:
I though we would be clearing to authorization cache too, although I forgot to put that into the card. Whaddayathink?